### PR TITLE
Ec collab

### PR DESCRIPTION
**Pull request by cherick04**
Please review this, @casey-collab

### CODE REVIEW

File: english/revolution/choices/Clinton.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You have chosen to accept the anhilation of your people.
```
Comment (casey-collab) [inline english/revolution/choices/Clinton.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cherick04** [comment]
Please review this, @casey-collab

**Comment by cherick04** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**